notebooks/NB_FINAL.ipynb

```diff
@@ -1110,13 +1110,15 @@
    "id": "9ae61585-5e2f-4e5f-b81b-4f94af6662a6",
    "metadata": {},
    "source": [
-    "### Treinando e Testando o modelo\n",
+    "### Treinando e Testando o Modelo\n",
     "\n",
-    "Neste estudo, adotamos o algoritmo Random Forest para nossa análise de regressão.\n",
-    "O algoritmo que escolhemos foi o **Random Forest**. O Random Forest é um algoritmo de aprendizado de máquina que é amplamente utilizado tanto para tarefas de regressão quanto para classificação. É uma técnica de ensemble, o que significa que combina várias árvores de decisão para obter previsões mais robustas e precisas.\n",
+    "Realizada a análise exploratória de dados, discutimos sobre qual seria a variável dependente (`y`), ou seja, o valor que desejamos predizer através do uso de algoritmos de aprendizado de máquina. Decidimos por focar nosso estudo em predizer o número de casos notificados, representado pela coluna `casos_notif`.\n",
     "\n",
-    "SUGESTÃO:\n",
-    "*Neste estudo, empregamos o algoritmo Random Forest em um contexto de modelagem de regressão. O Random Forest é uma técnica amplamente reconhecida e utilizada para problemas de regressão. Sua capacidade de combinar várias árvores de decisão resulta em previsões robustas e precisas para variáveis contínuas. A escolha desse algoritmo baseia-se em sua eficácia comprovada na tarefa de prever valores numéricos a partir de dados complexos.*"
+    "#### O Algoritmo Random Forest\n",
+    "\n",
+    "Ao enfrentar o desafio de prever o número total de casos notificados (`casos_notif`), optamos por utilizar o algoritmo Random Forest, uma escolha amplamente reconhecida em problemas de regressão. Este algoritmo é especialmente eficaz quando a variável a ser prevista é contínua, como é o caso da nossa aplicação, em que buscamos predizer o valor numérico representando o total de casos notificados.\n",
+    "\n",
+    "O Random Forest é um algoritmo mplamente utilizado tanto para tarefas de regressão quanto para classificação. Entretanto, pelo fato de a variável que desejamos predizer com o algoritmo ser contínua (valor numérico representando o total de casos notificados), utilizamos o Random Forest para executar a tarefa de regressão. Sua capacidade de combinar várias árvores de decisão resulta em previsões robustas e precisas para variáveis contínuas. A escolha desse algoritmo baseia-se em sua eficácia comprovada na tarefa de prever valores numéricos a partir de dados complexos.\n"
```

Comment:
The explanation is somewhat redundant; please change to:
 
"Ao abordar a previsão do número total de casos notificados (casos_notif), escolhemos o algoritmo Random Forest, amplamente reconhecido para problemas de regressão. Este algoritmo é especialmente eficaz para variáveis contínuas, como no nosso caso, onde buscamos prever o valor numérico que representa o total de casos notificados. A escolha do Random Forest para esta tarefa se baseia em sua capacidade comprovada de combinar várias árvores de decisão, resultando em previsões robustas e precisas para variáveis contínuas."